### PR TITLE
Exomast Retrieval

### PR DESCRIPTION
**Pull request by mfixstsci**
Pushing UI changes and some backend changes. Need to parse the result of ExoMAST retrieval back into `newHST` form.

### CODE REVIEW

**Comment by mfixstsci** [comment]
@bourque

**Comment by mfixstsci** [comment]
@nespinoza, just so you can see the progress

File: pandexo/engine/run_online.py [hunk 1]
```diff
@@ -502,21 +499,19 @@ def post(self):
...
             exodata["telescope"] = 'hst'
 
-
             #planet properties 
             properties = self.get_argument("properties")
-            
-            if properties=="user":
 
+            if properties=="user":
+                print('bad')
```
Comment (natashabatalha) [inline pandexo/engine/run_online.py]: Is this print statement supposed to be here?

**Comment by natashabatalha** [comment]
I just checked out your pull request and although the button is now there, it doesn't look like the button has an action paired to it. So naturally, it breaks.  What is the expected behavior of your pull request??

**Comment by mfixstsci** [comment]
Currently running the tool, I type a planet name in say, `wasp19b` and then click resolve target. The page reloads and all of the parameters are loaded in the `user defined properties` section.

File: pandexo/engine/run_online.py (hunk 1 above)
Comment (mfixstsci) [reply pandexo/engine/run_online.py]: Sorry, was using that for testing.

**Comment by natashabatalha** [comment]
Upon "resolve target" I get: 

> `    HTTPServerRequest(protocol='http', host='localhost:1111', method='POST', uri='/calculation/newHST', version='HTTP/1.1', remote_ip='::1')
    Traceback (most recent call last):
      File "/Users/nbatalh1/.conda/envs/pandexo/lib/python3.7/site-packages/tornado/web.py", line 1697, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/Users/nbatalh1/Documents/codes/PandExo/pandexo/engine/run_online.py", line 609, in post
        fileinfo_plan = self.request.files['planFile'][0]
    KeyError: 'planFile'`

And no changes are made to the html page.

**Comment by mfixstsci** [comment]
@natashabatalha ah ha. It's because I am running in `debug = True`. Let me trying something that I just thought of and see if it works.

**Comment by mfixstsci** [comment]
@natashabatalha okay, so I needed to add a return before rendering the html. I also added the semi-major axis to the retrieval.  It should work now, let me know (fingers crossed).

File: pandexo/engine/run_online.py [hunk 2]
```diff
@@ -589,7 +589,7 @@ def post(self):
                     inc = planet_data['inclination']
 
                 exodata["planet"]["i"]          = inc
-                exodata["planet"]["ars"]        = planet_data['a/Rs'] 
+                exodata["planet"]["a"]        = planet_data['a/Rs']
```
Comment (natashabatalha) [inline pandexo/engine/run_online.py]: In general, changing the dictionary keys of exo dict or the inst dict will mess up the people using the Python API. 

https://natashabatalha.github.io/PandExo/tutorialhst.html#edit-stellar-and-planet-inputs

Comment (mfixstsci) [reply pandexo/engine/run_online.py]: I see, I can make the change.

Comment (natashabatalha) [reply pandexo/engine/run_online.py]: Awesome thanks! Great work on this!!

**Comment by mfixstsci** [comment]
Hey @natashabatalha! This PR is ready for review now. @nespinoza and I have made some pretty significant changes to the UI and some smaller changes to the back end. Reach out if you have questions and we are hoping to hear from you soon!

File: pandexo/engine/reference/exo_input.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
             "exopath" : "filename_planet",
             "w_unit": "Angs,um,nm,Hz,sec,cm",
             "f_unit": "rp^2/r*^2 or fp/r*",
-            "transit_duration": "in seconds",
+            "transit_duration": "in days",
```
Comment (natashabatalha) [inline pandexo/engine/reference/exo_input.json:26]: Why was this changed? Were any latter defaults changed?

Comment (natashabatalha) [reply pandexo/engine/reference/exo_input.json:26]: nvm see that it's just used for a place holder!

**Comment by natashabatalha** [comment]
@mfixstsci great work! All looks good. My one comment is the `exo_input.json` get's printed out for the user to explore all inputs :

https://natashabatalha.github.io/PandExo/tutorialjwst.html#setting-up-a-run

The `url` and `url_err` is more of a backend thing that should be entirely hidden from the UI. I left it in there for this round, but be cognizant of that for future inserts. If more keys need to be added, we should create a separate `online_config.json`, for instance.

**Comment by natashabatalha** [comment]
@mfixstsci oh and one last comment! I ended up pushing your changes directly to master. For the next PR, let's branch/merge those off of `dev`
